### PR TITLE
Using https://asset-packagist.org instead of fxp/composer-asset-plugin

### PR DESCRIPTION
**Pull request by krivochenko**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  |

### CODE REVIEW

**Comment by krivochenko** [comment]
Were tests broken by me or earlier?

**Comment by samdark** [comment]
Earlier.

**Comment by samdark** [comment]
@SilverFire, @cebe what do you think?

**Comment by SilverFire** [comment]
Related to https://github.com/yiisoft/yii2/pull/14025

I'm all for this change.

**Comment by samdark** [comment]
OK. @SilverFire could you verify that it's done properly and merge it then?

File: docs/guide/start-installation.md [hunk 1]
```diff
@@ -10,14 +10,11 @@ The minimum requirement by this project template is that your Web server support
...
-With Composer installed, you can then install the application using the following commands:
+With Composer installed, you can then install the application using the following command:
 
-    composer global require "fxp/composer-asset-plugin:^1.2.0"
     composer create-project --prefer-dist yiisoft/yii2-app-advanced yii-application
 
-The first command installs the [composer asset plugin](https://github.com/francoispluchino/composer-asset-plugin/)
-which allows managing bower and npm package dependencies through Composer. You only need to run this command
-once for all. The second command installs the advanced application in a directory named `yii-application`.
+This command installs the advanced application in a directory named `yii-application`.
```
Comment (cebe) [inline docs/guide/start-installation.md]: the documentation should still mention that `asset-packagist` is used and also briefly discuss the alternatives between the asset-plugin and asset-packagist.

Comment (krivochenko) [inline docs/guide/start-installation.md]: > This command installs the advanced application in a directory named yii-application. You can choose a different directory name if you want. 
> It uses asset-packagist for managing bower and npm package dependencies through Composer. Also you can use asset-plugin, as in earlier versions, but it works slowly.

Is it ok?

Comment (samdark) [inline docs/guide/start-installation.md]: Yes.

**Comment by cebe** [comment]
Thank you!

**Pull request closed without merging**